packages/bsp/xiaomi-lmi/zz-update-abl-kernel

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -ex
+machine_model=$(cat /sys/firmware/devicetree/base/model|tr '\0' '\n')
+case $machine_model in
+	"Redmi k30 pro (SMS)")
+		panel_type=sms
+		;;
+	"Redmi k30 pro (CSOT)")
+		panel_type=csot
```

Comment:
Model is define as `Redmi k30 pro (lmi)` in devicetree, I think this will not work.